framework/api_vulkan_sample.cpp

```diff
@@ -860,7 +860,7 @@ void ApiVulkanSample::update_render_pass_flags(uint32_t flags)
 void ApiVulkanSample::on_update_ui_overlay(vkb::Drawer &drawer)
 {}
 
-void ApiVulkanSample::store_shader(const vkb::ShaderSourceLanguage& shaderLanguage, const std::vector<std::pair<vkb::ShaderType, std::string>>& listOfShader)
+void ApiVulkanSample::store_shader(const vkb::ShaderSourceLanguage &shaderLanguage, const std::vector<std::pair<VkShaderStageFlagBits, std::string>> &listOfShader)
```

Comment:
Naming conventions: `shaderLanguage` -> `shader_language`, and `listOfShader` -> `list_of_shaders`

Reply:
Done.